templates/web/base/admin/reportextrafields/edit.html

```diff
@@ -0,0 +1,68 @@
+[% INCLUDE 'admin/header.html' title=loc('Extra Fields') -%]
+
+<form method=post action="[% c.uri_for('', extra.id || 'new' ) %]">
+    <div class="admin-hint">
+      <p>
+        [% loc('Give this collection of fields a name. It is not shown publicly, just here in the admin.') %]
+      </p>
+    </div>
+    <p>
+        <label>
+            [% loc('Name') %]
+            <input type=text name="name" value="[% extra.name | html %]" />
+        </label>
+    </p>
+
+    [% IF cobrands.size > 1 %]
+        <div class="admin-hint">
+        <p>
+            [% loc('To limit this collection of fields to a single cobrand, select it here.') %]
+        </p>
+        </div>
+        <p>
+            <label>
+                [% loc('Cobrand') %]
+                <select name="cobrand">
+                    <option value="">[% loc('All cobrands') %]</option>
+                [% FOREACH cobrand IN cobrands %]
+                <option value="[% cobrand | html %]" [% IF cobrand == extra.cobrand %]selected[% END %]>[% cobrand | html %]</option>
+                [% END %]
+                </select>
+            </label>
+        </p>
+    [% ELSE %]
+        <input type=hidden name=cobrand value="[% extra.cobrand | html %]" />
+    [% END %]
+
+    [% IF languages.values.size > 1 %]
```

Comment:
No issue, just to note that `languages.size` would work on a hash and returns the same answer as `languages.values.size`.